NSS-Day-Cohort-69/tabloid-one-more-div
# Md feature/37/create a category

### **Purpose**
To allow a user with the role admin to create a category.

### **Files Changed**
- Created file, and added the class with the needed properties in new `CatergoryCreateDTO.cs`
- Added the endpoint for creating a new Category in `CategoryController.cs`
- Added the fetch for creating a new Category in `categoryManager.js`
- Added routing for Categories Create in view in `ApplicationView.jsx`
- Added nav button to Categories Create in view for logged in Admins in NavBar.jsx

### **To Test**
Fetch,setup, and run according to project README, then confirm the following

- Logged in Admin can click Create a Category on the Categories Management view
- Logged in Admin can Fill in the input bar
- Logged in Admin can create a Category

closes #37
